is_core/forms/forms.py

```diff
@@ -39,6 +41,7 @@ def as_widget(self, widget=None, attrs=None, only_initial=False):
             widget.is_localized = True
 
         attrs = attrs or {}
+        attrs = self.build_widget_attrs(attrs, widget)
```

Comment:
Tuhle metodu vůbec neznám, ale je to i v kódu Djanga u BoundField, tak jsi to asi udělal podle toho.